lib/modules/manager/npm/extract/locked-versions.ts

```diff
@@ -60,16 +60,16 @@ export async function getLockedVersions(
       if (lockfileVersion === 1) {
         if (packageFile.constraints?.npm) {
           // Add a <7 constraint if it's not already a fixed version
-          if (!semver.valid(packageFile.constraints.npm)) {
+          if (semver.satisfies('6.14.18', packageFile.constraints.npm)) {
```

Comment:
should we allow renovate to update that to latest minor and patch if there is an update available in future ( eg security update)?

Reply:
Sure, but let's do that in a future PR if someone feels strongly. For now this should solve 99% and there's no major harm done for the "else" case